spring-integration-zookeeper/src/main/java/org/springframework/integration/zookeeper/metadata/Conversions.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.zookeeper.metadata;
+
+import java.io.UnsupportedEncodingException;
+
+/**
+ *
+ * Utilities for null-safe conversions.
+ *
+ * @author Marius Bogoevici
+ */
+public class Conversions {
```

Comment:
Maybe we can move this to StringUtils, although the specific role of this class is to isolate null-safe `byte[]` <-> `String` conversions.